mkdocs.yml

```diff
@@ -16,33 +16,33 @@ theme:
 # Pages
 nav:
   - Welcome: "index.md"
-  - Introduction: "01_introduction.md"
-  - Project structure: "20_project_structure.md"
-  - Schema: "30_schema.md"
+  - Introduction: "introduction.md"
+  - Project structure: "project-structure.md"
+  - Schema: "schema.md"
   - User Guide:
-      - Create a research protocol: "41_create_new_protocol.md"
-      - Adopt assessments from the library: "42_adopt_assessments.md"
-      - Create new assessments for a protocol: "43_create_new_assess.md"
-      - Add a feedback section: "44_setup_feedback.md"
-      - Finalize the protocol: "45_finalize_protocol.md"
-      - Toolkit: "46_tools.md"
+      - Create a research protocol: "user_guide/create-new-protocol.md"
```

Comment:
on this page, we have `_` and `-` in filenames, maybe we want to be consistent to use one of them only?

Reply:
yeah I was not sure about this: I used `-` in filenames because [remark](https://github.com/remarkjs/remark) was complaining about it when I run it on the repo

if you prefer full consistency I can change it